docs-src/concepts/what-is-signal-with-start.md

```diff
@@ -0,0 +1,27 @@
+---
+id: what-is-signal-with-start
+title: What is Signal-With-Start?
+sidebar_label: Signal-With-Start
+description: Signal-With-Start starts and Signals a Workflow Execution, or just Signals it if it already exists.
+tags:
+  - term
+  - explanation
+---
+
+Signal-With-Start is a Client method that takes the following arguments:
+
+- A Workflow Id
+- Workflow input
+- A [Signal](/concepts/what-is-a-signal) name
+- Signal input
+
+If there is a running Workflow Execution with the given Workflow Id, it will be Signaled.
+Otherwise, a new Workflow Execution will be started and immediately Signaled.
+
+How to Signal-With-Start in:
+
+- [Go](/dev-guide/go/features#signal-with-start)
+- [Java](/dev-guide/java/features#signal-with-start)
+- [PHP](/dev-guide/php/features#signal-with-start)
+- [Python](/dev-guide/python/features#signal-with-start)
+- [TypeScript](/dev-guide/typescript/features#signal-with-start)
```

Comment:
Currently our style is to have the full question be linked:
- How to Signal-With-Start using the Go SDK 

for example

Reply:
I like that this is fewer words, feels easier to parse/read to me. Feels related to Aarohi's study result of people skipping over words/prose—if we use fewer words, maybe they're less likely to skip. Wdyt of changing style? 

<img width="807" alt="image" src="https://github.com/temporalio/documentation/assets/251288/56666acb-3b9c-4fcb-9b91-874608c1b825">


vs

<img width="797" alt="image" src="https://github.com/temporalio/documentation/assets/251288/78832c14-30c4-4b46-a9d3-3a38482ef230">